src/config/default-config.js

```diff
@@ -1,15 +1,28 @@
 const dotenv = require('dotenv');
 const AWS = require('aws-sdk');
+const fetch = require('node-fetch');
 const getLogger = require('../utils/getLogger');
 
 const logger = getLogger();
 
-// If we are not deployed on GitLab (AWS/k8s), the environment is given by
+
+let githubOrganisationName = 'hms-dbmi-cellenics';
+const getGithubRepo = async () => {
+  const answer = await fetch(`https://raw.githubusercontent.com/${githubOrganisationName}/iac/master/releases/production/pipeline.yaml`);
+  const text = await answer.text();
+  if (text === '404: Not Found') {
+    githubOrganisationName = 'biomage-ltd';
+  }
+};
+getGithubRepo();
```

Comment:
I would change the `getGithubRepo` function to return the repo name and then make `const githubOrganisationName = getGithubRepo()`. The code will be much clearer this way. Also, I will rename `getGithubRepo` to `getGithubOrg`